apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/BooleanExpression.kt

```diff
@@ -125,14 +132,21 @@ sealed class BTerm
 
 /**
  * A term that comes from @include/@skip directives and that needs to be matched against operation variables
+ * TODO rename to BInclude
```

Comment:
Should we? Not 100% sure of the naming to be honest but "variable" sounds off to me.

Reply:
Ideally we can "uniformize" that. Something like this?

1. `@include(if: $var1)` = BVariable("var1")
2. `@skip(if: $var1)` = !BVariable("var1")
3. `@defer(label: $lbl1)` = !BLabel("lbl1")
4. `@defer(label: $lbl1, if: $var1)` = BLabel("lbl1") || !BVariable("var1")

Reply:
👍 Yes it makes sense! Done in e2e26ce772f89b80f055efc235ef86165acb663d.